web/single-object.html

```diff
@@ -159,6 +165,10 @@ <h5 class="modal-title mx-auto" id="shareModalLabel">Share {{.Data.Key}}</h5>
   }
 
   window.onload = async function () {
+      const fileSize = {{.Data.SizeNumber}} || 0;
+      const FOUR_KB = 4096;
+      const isLittleFile = fileSize < FOUR_KB
```

Comment:
Tying this information to content size seems wrong, since the specific threshold may change over time.

Reply:
@egonelbre  that's true but even message copy is tied to content size. So if we want to omit using content size here we should change a copy too?

Reply:
is there any way to get this threshold in a code?

Reply:
The new statellite returns `piece_count`... if that's 0, then you can assume it's stored on the satellite.

Reply:
ok, by the way, should we display pieceCount or reliablePieceCount? Values are different. 
f.e. https://link.tardigradeshare.io/s/15CYbbCAnFyMjXZdZs8sVJ8JbGdCKjdaMCjhNKF937pUMikZnXCBDu6SeemVbQ1opJW257otj7SXFaM7DSEGf9Fm6LjtSPQycMXskWbsbWyNY6q6vetvhRY8TsBLKxuB5778sNfVHvnstGHRUB26cKNpZsnRjCXfzqyDLCB2Rd2euu3qf4d3sNFHKwrym9Ubf8ctbAUsQyruK7TJhFHVBGgQLfVDXnoHZo7CcGXDE1gxL27ac2u8YN1tj4vAGvTVy5e2V9mArg7wrJ3q25g9WwerDjScgQFwvPTABBjwyckH8dmH6cTyhqfgsf8UMj9gAc8ygJHbrroZkx3PpxeEvLVHpGvoWSQbyR7SDqdp1UYQ5gCG1T4nPoRrxjMHDCwgScEAAoBb7TJu5pmznEk6KEXb98oriJ1NU3Gh7oykbeQCKHMnwNd2e8W1NcMbFJN8TrWtpzDXafQSWDPEykJot3o3526VWYBDXdCao3C31V99mMf2tEcaWFDnhjv9a97Ck2UKZ8EB8vwPq4tW/homepage/TardigradeExplainerVideo.m4v

for this object we've got 80 pieceCount, 42 reliablePieceCount, 42 ports and 1 segmentCount

Reply:
Let's show the `pieceCount` rather than `reliablePieceCount`, because otherwise the number of pieces keeps changing.

Reply:
I disagree, we should show the live count (meaning reliable). It reflects the reality and where you actually pick the pieces from.

Reply:
Fair enough, but in that case we should have a clear explanation that this value may change from hour to hour.

Reply:
done

Reply:
I just discovered a bug in how reliable piece count is determined, let's hide it for now, such that people won't panic when they see a small number.

Reply:
ok, so now we show `pieceCount` without any message